### ISSUE TITLE
Vue 2.0 

### ISSUE THREAD

**Issue by Ben52** [vue-bulma/datepicker#6]
I'm trying to install in a Vue 2.0 project, and getting 
`Peer vue-bulma-datepicker@1.1.0 wants vue@^1.0.26` message.

**Comment by fundon** [comment]
Currently, it's not published to NPM. So use this `npm i vue-bulma/datepicker`.

**Comment by projct1** [comment]
I try this: `Vue.component('datepicker', require('vue-bulma-datepicker'));`
And have error:
`Module not found: Error: Can't resolve './WrapperInput' in 'D:\WEB\domains\bron\node_modules\vue-bulma-datepicker\src'`

**Comment by fundon** [comment]
@rorc Which vue version?

**Comment by projct1** [comment]
@fundon 2.0.3

**Comment by fundon** [comment]
@rorc Do you use webpack?

**Comment by projct1** [comment]
@fundon Yep =)

**Comment by fundon** [comment]
I'm on macOS, So I should find a windows for testing. But the [appveyor ci](https://ci.appveyor.com/project/fundon/vue-admin) is green.

**Comment by silentdragonz** [comment]
I had the same issue along with a stylus error. I had to install stylus and stylus-loader which weren't required anywhere or mentioned that I had to have them installed in the docs or from NPM.

To fix the SingleInput/WrapperInput errors I had to add `resolve: {
                extensions: ['', '.js', '.vue'],
        }
` to my webpack config to make it work. If you include the .vue on your imports it'll probably be more compatible with people who don't have the above code in their configs. This is the first time I've ever seen it.
